src/main/java/com/example/Employee_Management_System/Repositories/EmployeeRepository.java

```diff
@@ -7,11 +7,16 @@
 
 import java.util.List;
 
-public interface EmployeeRepository extends JpaRepository<Employee,Integer> {
+public interface EmployeeRepository extends JpaRepository<Employee, Integer> {
 
-//    @Query("select e from Employee e where e.name = ?1")
-        @Query("select e from Employee e where e.name like %:keyword%")
-      List<Employee> findByName( @Param("keyword") String name);
+    @Query("select e from Employee e where e.name like %:keyword%")
+    List<Employee> findByName(@Param("keyword") String name);
 
+    //To check whether the email entered is already present or not
+    boolean existsEmployeeByEmail(String email);
 
+    boolean existsEmployeeByPhoneNumber(String phoneNumber);
+
+    @Query(value = "select * from employee where (email=?1 or phone_number=?2) and id <>?3", nativeQuery = true)
+    List<Employee> updatedEmailOrPhoneNumberExists(String email, String phone, int id);
```

Comment:
rename updatedEmailOrPhoneNumberExists to findByEmailOrPhoneNumberAndNotEqualsId